app/src/main/res/layout/topic_train_skill_view.xml

```diff
@@ -13,20 +13,23 @@
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
     android:layout_height="wrap_content">
+
     <CheckBox
       android:id="@+id/skill_check_box"
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
       android:layout_margin="4dp"
+      android:button="@drawable/selector_checkbox"
       android:checked="@{isChecked}"
-      android:drawablePadding="4dp"
+      android:fontFamily="sans-serif"
       android:gravity="center_vertical"
+      android:paddingLeft="@dimen/padding_checkbox"
```

Comment:
Please use paddingEnd instead of paddingLeft for a RTL-compatible layout.

Reply:
Done.